src/bindings/python/src/openvino/frontend/pytorch/ts_decoder.py

```diff
@@ -85,7 +86,12 @@ def _get_scripted_model(self, pt_module, example_inputs=None, skip_freeze=False)
                 scripted = torch.jit.script(pt_module)
             else:
                 input_parameters, input_signature, pt_module, self._input_is_list = prepare_example_inputs_and_model(example_inputs, input_params, pt_module)
-                scripted = torch.jit.trace(pt_module, **input_parameters, strict=False)
+                try:
+                    gptq.patch_model(pt_module)
+                    scripted = torch.jit.trace(
+                        pt_module, **input_parameters, strict=False)
+                except:
+                    gptq.unpatch_model(pt_module)
             if not skip_freeze:
                 for n in scripted.inlined_graph.nodes():
```

Comment:
`scripted` variable is no longer accessible here if you get an exception in the logic above.